validation/eskipvalidator.go

```diff
@@ -0,0 +1,18 @@
+package validation
+
+import (
+	"github.com/zalando/skipper/eskip"
+	"github.com/zalando/skipper/filters"
+	"github.com/zalando/skipper/routing"
+)
+
+type EskipValidator interface {
+	ValidateFilters(ctx ResourceContext, filters []*eskip.Filter) error
+	ValidatePredicates(ctx ResourceContext, predicates []*eskip.Predicate) error
+	ValidateRoute(ctx ResourceContext, routes []*eskip.Route) error
+	ValidateBackend(ctx ResourceContext, backend string, backendType eskip.BackendType) error
+}
+
+type Runner interface {
```

Comment:
Why do you need this to be an interface? If we remove this nothing will fail and we don't have multiple runners